skins/plantswap.scss

```
@@ -0,0 +1,10 @@
+.my-space {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.plantswap {
```